doc/theoretical_description_calibrators.rst

```diff
@@ -0,0 +1,80 @@
+.. title:: Calibrators : contents
+
+.. _theoretical_description_calibrators:
+
+###############
+Calibrators
+###############
+
+In Mapie, the conformalisation step is done directly inside
+:class:`~mapie.regression.MapieRegressor` or :class:`~mapie.classification.MapieClassifier`,
+depending on the ``method`` argument.
+However, when implementing the new CCP method, we decided to externalize the conformalisation
+step into a new object named ``calibrator``, to have more freedom and possible customisation.
+
+The new classes (:class:`~mapie.futur.split.SplitCPRegressor` and :class:`~mapie.futur.split.SplitCPClassifier`) have 3 steps:
+
+1. ``fit_predictor``, which fit the sklearn estimator
+2. ``fit_calibrator``, which do the conformalisation (calling ``calibrator.fit``)
+3. ``predict``, which compute the predictions and call ``calibrator.predict`` to create the prediction intervals
+
+Thus, the calibrators, based on :class:`~mapie.calibrators.base.BaseCalibrator`,
+must have the two methods: ``fit`` and ``predict``.
+
+Mapie currently implements calibrators for the CCP method (and the naive method),
+but any conformal prediction method can be implemented by the user as
+a subclass of :class:`~mapie.calibrators.base.BaseCalibrator`.
+
+Example of naive Split CP:
```

Comment:
Why taking example of the naive split CP ? (naive here means that you don't have coverage), so actually it's not a CP method

Reply:
You are right, I changed it